tests/utils/callbacks/visualizer_callback/test_visualizer.py

```diff
@@ -34,8 +34,10 @@ def test_add_images(dataset):
         )
         logger = get_dummy_logger(config, dir_loc)
         model = get_dummy_module(config)
-        trainer = pl.Trainer(callbacks=model.callbacks, logger=logger, checkpoint_callback=False)
+        callbacks = [VisualizerCallback(loggers=["local", "tensorboard"])]
```

Comment:
In line 51, of `dummy_lightning_module.py` we initialize VisualizerCallback. I feel we should move like 37 in this file to line 51 and use `Trainer(callbacks=model.callbacks..`